middleware/compress_test.go

```diff
@@ -89,7 +89,9 @@ func TestGzipWithStatic(t *testing.T) {
 		assert.Equal(t, cl, rec.Body.Len())
 	}
 	r, err := gzip.NewReader(rec.Body)
-	assert.NoError(t, err)
+	if !assert.NoError(t, err) {
```

Comment:
Can you move code into assert.NoError() block like https://github.com/ferhatelmas/echo/blob/6436430d464309275ea5218c1e30b6e81c1d4137/middleware/compress_test.go#L38